File: stringresult02.c
#include <stdio.h>
#include <string.h>

void readStrings(char strings[10][100]) {
    printf("Enter 10 strings:\n");

    for (int i = 0; i < 10; i++) {
        scanf("%s", strings[i]);
    }
}

void concatenateStrings(char strings[10][100], char result[1000]) {
    strcpy(result, strings[0]);

    for (int i = 1; i < 10; i++) {
        strcat(result, " ");
        strcat(result, strings[i]);
    }
}

int main() {
    char strings[10][100];
    char result[1000];

    readStrings(strings);
    concatenateStrings(strings, result);

    printf("Concatenated string: %s\n", result);

    return 0;
}

File: stringresult02.py
def read_strings():
    strings = []
    print("Enter 10 strings:")
    for i in range(10):
        strings.append(input())
    return strings

def concatenate_strings(strings):
    return " ".join(strings)

def main():
    strings = read_strings()
    result = concatenate_strings(strings)
    print("Concatenated string:", result)

if __name__ == "__main__":
    main()

